code/src/pages/MovieList.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from "react"
+import { Link } from "react-router-dom"
+
+import { MOVIES_URL } from "reusable/urls"
+
+export const MovieList = () => {    
+  const [movies, setMovies] = useState([])
+
+  useEffect(() => {
+    fetch (MOVIES_URL)
+    .then(res => res.json())
+    .then(json => setMovies(json.results))  
+    .catch(err => console.error(err))   
```

Comment:
might be a taste of ours but indentation could be added for these three lines when they are coming after a fetch() 😇